ykrt/build.rs

```diff
@@ -1,6 +1,62 @@
+use regex::Regex;
 use std::env;
+use std::os::unix::fs::symlink;
+use std::path::PathBuf;
+use std::process::Command;
+
+include!("../build_aux.rs");
+
+const YKSHIM_SO: &str = "libykshim.so";
 
 fn main() {
+    let ykrt_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    let mut internal_dir = ykrt_dir.clone();
+    internal_dir.push("..");
+    internal_dir.push("internal_ws");
+    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+    let cargo = env::var("CARGO").unwrap();
+    let rust_flags = env::var("RUSTFLAGS").unwrap_or_else(|_| String::new());
+    let tracing_kind = find_tracing_kind(&rust_flags);
+
+    // Only build the internal workspace now if we are not using xtask, i.e. if we are being
+    // consumed as a dependency. This means we see the compiler output for the internal workspace
+    // correctly when working directly with the yk repo (via xtask).
+    if env::var("YK_XTASK").is_err() {
+        env::set_current_dir(&internal_dir).unwrap();
+        let status = Command::new(cargo)
+            .arg("build")
+            .arg("--release")
+            .env_remove("RUSTFLAGS")
+            .env(
+                "RUSTFLAGS",
+                &format!("--cfg tracermode=\"{}\"", tracing_kind),
+            )
+            .spawn()
+            .unwrap()
+            .wait()
+            .unwrap();
+        if !status.success() {
+            panic!("building internal workspace failed");
```

Comment:
```suggestion
            eprintln!("building internal workspace failed");
            std::process::exit(1);
```

Avoids a backtrace when `RUST_BACKTRACE=1` is set.